### ISSUE TITLE
Not working anymore...

### ISSUE THREAD

**Issue by DarthSonic** [microsoft/azure-pipelines-jira#45]
...Azure Pipeline for Jira is not available any more on Atlassian Marketplace
...if installed before it was removed, it can not be configured anymore, organization cannot be added anymore
...if configured before removal, deployments are not added to jira, no event is triggered by Azure DevOps release pipeline
...also organization cannot be removed from configuration

As there are so many users relating on this app, please fix all that issues. There is no alternative for that. Service Hooks are not able to send a proper data payload to Jira Automation Web Hook Url. And all other Automation Tools do not support Azure Release trigger.

**Comment by ruslan-oncoshot** [comment]
Same issue here, suddenly the integration stopped working. As you can see on the picture June 22nd last time when Jire received deployment update from Azure DevOps, after that - nothing. 
![image](https://user-images.githubusercontent.com/84002608/178643031-2df1918d-2c10-4929-a19d-380f6fd0b36c.png)

**Comment by alrosmarz** [comment]
Same here, it stopped working on June 28 for us, after that no more deployment notifications were sent to JIRA, tried to configure it again but the plugin is not available in the atlassian marketplace :(

**Comment by marncosta** [comment]
I found it on atlassian marketplace but it is not working anymore. It does not appear on Azure Devops service connection page.

**Comment by DarthSonic** [comment]
When will this be fixed?

**Comment by milanm** [comment]
Yes, seems that its not working anymore and also not working for yaml pipelines too.

**Comment by benbward** [comment]
An acknowledgement that they're abandoning it would be nice.
